client/my-sites/plugins/plans/index.tsx

```diff
@@ -66,6 +68,36 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 					isReskinned
 				/>
 			</div>
+
+			<div className="plugin-plans-step-hints">
```

Comment:
Can you use the [Section component](https://github.com/Automattic/wp-calypso/blob/e99173ff814799c18d52140af9ced68f179ce870/client/my-sites/plugins/education-footer/index.tsx#L47) this?

You can make the headers optional if they aren't currently. Just want to get all the separation margins handled by a single component.

Reply:
Good to know @lsl, [as we replaced the code using the PromoSection](https://github.com/Automattic/wp-calypso/pull/68058/commits/c2656742a1acbe033b96483752b2d5b44fa2b315), I don't think we'll need this anymore.